src/main/java/ee/kertmannik/quiz/client/QuizException.java

```diff
@@ -0,0 +1,8 @@
+package ee.kertmannik.quiz.client;
+
+public class QuizException extends Exception {
```

Comment:
I think it is more convenient to extend RuntimeException, because then you don't have to write "throws" in every method signature. Yes, it may seem like good way to track exceptions in very small apps, but it tends to get out of hand in real-world apps. So fine to di it here, but generally can go with RunitemException.